src/main.ml

```diff
@@ -45,7 +45,8 @@ module SM = Storage_interface.ClientM(struct
       return (Jsonrpc.response_of_string result)
   end)
 
-let uri = ref "http://127.0.0.1/"
+(** Xapi's local Unix domain socket *)
+let uri = "file:///var/xapi/xapi"
```

Comment:
There is an entry `unix_domain_socket` in `xapi_globs.ml` - how are these related? If we need a new path, should this not be defined in `xapi_globs.ml`, too, rather than here?

Reply:
Good point. I've looked at the `_oasis` file of xapi, and it seems that `Xapi_globs` isn't exposed as a library. But I agree that we should have this constant in a publicly available xapi library and in `xen-api-client`.

Reply:
I forgot that this is outside of xapi. So maybe it is fine not to wait for xapi to expose its constants to libraries. This project, however, could have a dedicated module that contains such constants rather than spread them across the code. This will also make it easier to import them for a future library or read them from a file.

Reply:
We could pass it as a CLI parameter and use the value above as a default. This would give us some time to update xapi as well